src/transformers/adapter_training.py

```diff
@@ -1,70 +1,44 @@
 from dataclasses import dataclass, field
 from typing import Optional
 
-from .adapter_bert import BertModelHeadsMixin
-from .adapter_config import AdapterConfig, AdapterType
-
 
 @dataclass
 class AdapterArguments:
     """
     The subset of arguments related to adapter training.
     """
 
-    train_adapter: bool = field(
-        default=False, metadata={"help": "Train a text task adapter instead of the full model."}
+    train_adapter: bool = field(default=False, metadata={"help": "Train an adapter instead of the full model."})
```

Comment:
I find the train_adapter slightly confusing because I would expect similar behavior to "load_adapter": ie that this field is a list of adapter names that should be trained. the others remain fixed. 

We should probably keep it for now because changing the naming is a breaking change and we dont have a solution for the versioning problem

Reply:
agree, probably something like `--do_train_adapter` or `--do_use_adapter` would be better.